stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/channel/list/ChannelItem.kt

```diff
@@ -148,12 +150,14 @@ public fun DefaultChannelItem(
  *
  * @param channel The channel to show the info for.
  * @param currentUser The currently logged in user, used for data handling.
+ * @param isMuted If the channel is muted for the current user.
  * @param modifier Modifier for styling.
  */
 @Composable
 public fun ChannelDetails(
     channel: Channel,
     currentUser: User?,
+    isMuted: Boolean = false,
     modifier: Modifier = Modifier,
```

Comment:
We should put the modifier as the first non-optonal argument here if we want to keep the `isMuted` as false by def!